backend/src/db/mongoose.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const initDb = () => mongoose.connect(process.env.DATABASE_URL, {
```

Comment:
You can start the return from next line. It will be easier to read and would be indented well :)

Reply:
If your function just returns, and you use return keyword ESLINT shows error. Or there is something else which I am understading differently from what you are trying to say?